superset-frontend/src/views/CRUD/welcome/Welcome.tsx

```diff
@@ -180,6 +180,8 @@ function Welcome({ user, addDangerToast }: WelcomeProps) {
   };
 
   const WelcomeTopExtension = extensionsRegistry.get('welcome.banner');
+  const WelcomeDataExtension = uiOverrideRegistry.get('welcome.data');
+  const WelcomeTableExtension = uiOverrideRegistry.get('welcome.table');
```

Comment:
Is it necessary to have two extension points here? From the Superset perspective, this extension is a general purpose "replace the homepage with anything" component, that Preset happens to be using for some data and a table.